infra/modules/providers/azure/storage-account/variables.tf

```diff
@@ -0,0 +1,62 @@
+variable "resource_group_name" {
+  description = "The name of the resource group."
+  type        = string
+}
+
+variable "resource_group_location" {
+  description = "The location of the resource group."
+  type        = string
+}
+
+variable "account_name" {
+  description = "The name of the storage account service."
+  type        = string
+}
+
+variable "storage_container_names" {
+  description = "The list of storage container names to create. Names must be unique per storage account."
+  type        = list(string)
+}
+
+variable "performance_tier" {
+  description = "Determines the level of performance required."
+  type        = string
+  default     = "Standard"
+}
+
+variable "replication_type" {
+  description = "Defines the type of replication to use for this storage account. Valid options are LRS*, GRS, RAGRS and ZRS."
+  type        = string
+  default     = "LRS"
+}
+
+variable "kind" {
+  description = "Storage account types that determine available features and pricing of Azure Storage. Use StorageV2 when possible."
+  type        = string
+  default     = "StorageV2"
+}
+
+variable "https" {
+  description = "Boolean flag which forces HTTPS in order to ensure secure connections."
+  type        = string
```

Comment:
Should this be typed as bool?

Reply:
Thanks for pointing this out! This variable now uses bool as a type.